Makefile

```diff
@@ -0,0 +1,51 @@
+artifact_name       := chips-rest-interfaces-consumer
```

Comment:
Please add this target to the makefile: https://github.com/companieshouse/ci-concourse-resources/tree/master/tasks/java/maven/security-check#makefile

It will allow us to run dependency check in the pipeline against this repo. Similar to npm audit

Reply:
Put in the dependency check after clean